### ISSUE TITLE
netCDF reader and auto domain decomposition 

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1841]
When reading a netcdf with the basic reader in parallel the domain decomposition works but only one domain gets rendered.

This occurs with Pseudocolor and vector plots. It also prevents integral curves from being advected.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Allen Sanderson
Original creation: 04/03/2015 03:29 pm
Original update: 04/07/2015 06:53 pm
Ticket number: 2207
